src/components/AddedCourses/AddedCoursePane.js

```diff
@@ -101,12 +104,18 @@ class AddedCoursePane extends PureComponent {
         this.setState({ customEvents: AppStore.getCustomEvents() });
     };
 
+    loadScheduleNames = () => {
+        this.setState({ scheduleNames: AppStore.getScheduleNames() });
+    };
+
     getGrid = () => {
         return (
```

Comment:
Let's pull this out to a variable outside of the render, so the jsx is cleaner.
```suggestion
    getGrid = () => {
        const scheduleName = this.state.scheduleNames[AppStore.getCurrentScheduleIndex()]
        const scheduleUnits = this.state.totalUnits
        return (
```